extensions/amp-instagram/0.1/amp-instagram.js

```diff
@@ -76,16 +82,43 @@ class AmpInstagram extends AMP.BaseElement {
   }
 
   /** @override */
-  prerenderAllowed() {
-    return true;
+  prerenderCallback() {
+    const image = new Image();
```

Comment:
So, the same question here: what if we moved this as a placeholder and built it in the `buildCallback`?